Gitagram/View/MatchingView/Component/CardView.swift

```diff
@@ -91,7 +95,12 @@ private extension CardView {
 private extension CardView{
     
     func openLink(url: String){
-        openURL(URL(string: url)!)
+        if url != ""{
+            openURL((URL(string: url))!)
+        }else{
+            isShowAlert.toggle()
```

Comment:
[more]
ここの処理の意図はtoggleを切り替えるというよりは、アラートを出すという意図と伝えたいから、trueを代入する書き方にした方がわかりやすいかも。
```suggestion
            isShowAlert = true
```

Reply:
https://github.com/ProgateHackathon/Gitagram/pull/70/commits/1c3ab77b4ab19df97f25ea3317bcc693e4b85209
で修正しました！
@urassh